modules.d/77dracut-systemd/dracut-cmdline-ask.sh

```diff
@@ -18,7 +18,7 @@ echo "Enter additional kernel command line parameter (end with ctrl-d or .)"
 # shellcheck disable=SC3045
 while read -r -p "> " ${BASH:+-e} line || [ -n "$line" ]; do
     [ "$line" = "." ] && break
-    [ -n "$line" ] && printf -- "%s\n" "$line" >> /etc/cmdline.d/99-cmdline-ask.conf
+    [ -n "$line" ] && printf -- "%s\n" "$line" >> /etc/cmdline.d/20-cmdline-ask.conf
```

Comment:
These are not regular cmdline arguments from built-in modules. I wonder if they should remain at 99. Likewise for `dracut-cmdline` below.